citesphere/src/main/java/edu/asu/diging/citesphere/web/user/ItemController.java

```diff
@@ -33,8 +37,26 @@ public String getItem(Authentication authentication, Model model, @PathVariable(
             List<String> fields = new ArrayList<>();
             citationManager.getItemTypeFields((IUser)authentication.getPrincipal(), citation.getItemType()).forEach(f -> fields.add(f.getFilename()));
             model.addAttribute("fields", fields);
+            model.addAttribute("index", index);
+            model.addAttribute("page", page);
+            model.addAttribute("collectionId", collectionId);
+            model.addAttribute("sortBy", sortBy);
         }
         return "auth/group/items/item";
     }
     
+    @RequestMapping(value="/auth/group/{zoteroGroupId}/items/{itemId}/prevAndNext")
+    public @ResponseBody CitationPage getPrevAndNextItem(Authentication authentication, Model model, @PathVariable("zoteroGroupId") String zoteroGroupId, @PathVariable("itemId") String itemId,
+            @RequestParam(required = false, value = "index") String index, @RequestParam(defaultValue = "1", required = false, value = "page") int page,@RequestParam(value="collectionId", required=false) String collectionId,
+            @RequestParam(defaultValue = "title", required = false, value = "sortBy") String sortBy) throws GroupDoesNotExistException, CannotFindCitationException, ZoteroHttpStatusException {
+        CitationPage result = new CitationPage();
+        if(index != null) {
+            result = citationManager.getPrevAndNextCitation((IUser)authentication.getPrincipal(), zoteroGroupId, collectionId, page, sortBy, Integer.valueOf(index));
+            result.setIndex(index);  
+        }
+        result.setZoteroGroupId(zoteroGroupId);
+        result.setCollectionId(collectionId);
+        result.setSortBy(sortBy);
```

Comment:
all of these parameters are passed into `getPrevAndNextCitation`. They should be set there, not here. then you can return after line 55 and create a new citation page afterwards if index is null.